contrib/freebsd/distinfo

```diff
@@ -0,0 +1,3 @@
+TIMESTAMP = 1622115022
```

Comment:
I don't see any sed into this file, does it need some?

Reply:
There is no need, it's overwritten during the build. `make makesum` generates distinfo for the current commit:
https://github.com/fwupd/fwupd/pull/3274/commits/cd5c8eb8389f0094b1e5abe7ef94793c3047c7b4#diff-bdce2284300df67bb504fd10f5eb71470dbe6d3c074ff78fc6ccaaf405e6e008R54

Reply:
Then perhaps just drop the file alltogether if it's generated during make command.